basic-examples/expressions/workflow.yaml

```diff
@@ -1,25 +1,49 @@
----
 version: v0.2.0
 input:
   root: RootObject
   objects:
     RootObject:
       id: RootObject
-      properties: {}
+      properties:
+        name:
+          display:
+            description: Just a name
+            name: Name
+          required: true
+          type:
+            type_id: string
+        int_value:
+          display:
+            description: Just an integer value
+            name: Integer Value
+          required: true
+          type:
+            type_id: integer
+        float_value:
+          display:
+            description: Just a float value
+            name: Float Value
+          required: true
+          type:
+            type_id: float
+
 steps:
-  expression_loop:
-    kind: foreach
-    workflow: subworkflow.yaml
-    items:
-      - name: !expr '"Here''s an apostrophe and \"embedded quotes\"."'
-      - name: !expr "'Here\\'s an apostrophe and \"embedded quotes\".'"
-      - name: !expr |-
-          'Here\'s an apostrophe and "embedded quotes".'
-      - name: !expr |-
-          "Here's an apostrophe and \"embedded quotes\"."
-      - name: !expr '`Here''s an apostrophe and "embedded quotes".`'
-      - name: !expr |-
-          `Here's an apostrophe and "embedded quotes".`

```

Comment:
Is this available in another example?

I ask because this is a really critical resource, and it would be a significant loss to simply delete it:  quoting is _hard_, and these examples are an excellent resource for anyone wrestling with it.

Reply:
This is covered well in the existing documentation, and I don't think it serves as a very useful example here of how to use functions.

Reply:
Granted.  I just don't want to lose having this in machine executable form.